frontend/lib/with-apollo-client.tsx

```diff
@@ -46,10 +49,14 @@ const withApolloClient = (App: any) => {
 
       const apollo = initApollo(undefined, accessToken)
 
-      const { data } = await apollo.query({ query: UserDetailQuery })
+      // prevent repeating useroverview queries
+      if (accessToken && !currentUserCache[accessToken]) {
+        const { data } = await apollo.query({ query: UserDetailQuery })
+        currentUserCache[accessToken] = data.currentUser
+      }
 
```

Comment:
Wonder if this cache is okay - it just _might_ retain previous user info, but that's very unlikely?

Reply:
I think it does retain previous user info